mmv1/third_party/terraform/services/logging/resource_logging_project_sink_test.go

```diff
@@ -415,6 +448,66 @@ resource "google_storage_bucket" "log-bucket" {
 `, name, envvar.GetTestProjectFromEnv(), bucketName)
 }
 
+func testAccLoggingProjectSink_customWriter(name, bucketName string, serviceAccount string) string {
+	return fmt.Sprintf(`
+resource "google_service_account" "test-account1" {
+  account_id   = "%s"
+  display_name = "Log Sink Custom WriterIdentity Testing Account"
+}
+
+resource "google_project_iam_member" "custom-sa-logbucket-binding" {
+  project = "gchen-sharedvpc"
+  role   = "roles/logging.bucketWriter"
+  member = "serviceAccount:${google_service_account.test-account1.email}"
+}
+
+resource "google_service_account_iam_member" "loggingsa-customsa-binding" {
+	service_account_id = google_service_account.test-account1.name
+	role   = "roles/iam.serviceAccountTokenCreator"
+	member = "serviceAccount:service-%s@gcp-sa-logging.iam.gserviceaccount.com"
+}
+
+resource "google_logging_project_sink" "custom_writer" {
+  name        = "%s"
+  destination = "logging.googleapis.com/projects/gchen-sharedvpc/locations/us-central1/buckets/shared-logsink-bucket"
```

Comment:
Reference test project id.

```suggestion
  destination = "logging.googleapis.com/projects/${data.google_project.project.project_id}/locations/us-central1/buckets/shared-logsink-bucket"
```

Reply:
Fixed